### PR TITLE
react-validated-input-component | Daniel To

### PR DESCRIPTION
**Pull request by theDanielTo**
react-validated-input

### CODE REVIEW

File: react-validated-input-component/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  handleChange = e => {
+    this.setState({
+      password: e.target.value
+    });
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    if (this.state.password.length === 0) {
+      this.setState({ errorMsg: '' });
```
Comment (scbowler) [inline react-validated-input-component/src/index.jsx:27]: Here the error message should be something like: `"Password is required"`

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but there should be a message if the form is submitted without anything in the password field. Let me know if you have any questions

File: react-validated-input-component/src/index.jsx (hunk 1 above)
Comment (theDanielTo) [reply react-validated-input-component/src/index.jsx]: react-validated-input2

**Review by scbowler — APPROVED** [review]
Looks great and nice job on the extra challenge! 👍 👍

**Pull request merged**